vmdb/app/controllers/api_controller.rb

```diff
@@ -95,6 +96,13 @@ def initialize
     @api_token_mgr   = TokenManager.new(@module)
   end
 
+  def self.all_encrypted_attributes
+    (ATTR_TYPES[:encrypted]             |
+     ::Authentication.encrypted_columns |
+     ::MiqDatabase.encrypted_columns    |
```

Comment:
LOL I just noticed it said harboring.  Silly autocorrect.  I really don't like *hardcoding* these classes.